lib/utils/array.js

```diff
@@ -1,3 +1,21 @@
-export default function last(arr) {
+export function last(arr) {
   return arr[arr.length - 1];
 }
+
+export const arrBuf2Base64 = buffer => {
+  let base64Data = null;
+  try {
+    const binaryString = String.fromCharCode.apply(null, new Uint8Array(buffer));
+    base64Data = 'base64,' + window.btoa(binaryString);
```

Comment:
Better to use the [Buffer class](https://nodejs.org/docs/latest/api/buffer.html#buffer_buf_tostring_encoding_start_end), imo.
Something like: 
```
base64Data = Buffer.from(buffer).toString('base64')
```